schemas/context/destination.schema.json

```diff
@@ -0,0 +1,79 @@
+{
+  "meta:license": [
+    "Copyright 2019 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/destination",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Destinations",
+  "type": "object",
+  "meta:extensible": false,
+  "meta:abstract": false,
+  "meta:extends": [
+    "https://ns.adobe.com/xdm/data/record"
+  ],
+  "description": "Destinations Schema.",
+  "definitions": {
+    "destination": {
+      "properties": {
+        "xdm:destinationID": {
+          "title": "Destination ID",
+          "type": "string",
+          "description": "Unique Identifier for destination flow"
+        },
+        "xdm:destinationName": {
+          "title": "Destination Name",
+          "type": "string",
+          "description": "Destination flow name"
+        },
+        "xdm:destinationAccountName": {
```

Comment:
We don't need `destinationAccountName` anymore as noted [here](https://jira.corp.adobe.com/browse/PLAT-58327?focusedCommentId=21995568&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-21995568).